cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -121,6 +115,9 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     
     private boolean sslEnabled;
     private boolean authEnabled;
+    private boolean userManagedApiUsersEnabled;
```

Comment:
Do you really need this boolean? You can probably check it through the `userManagedApiSecretName` field for example and change the `isUserManagedApiUsersConfigEnabled` method to something like `validateApiUsersConfig`, or?

Reply:
You are right, I should be able to safely removed now. Just removed